src/Tribe/Timezones.php

```diff
@@ -126,45 +126,17 @@ public static function event_end_timestamp( $event_id, $timezone = null ) {
 	 *
 	 * @param int    $event_id
 	 * @param string $type (expected to be 'Start' or 'End')
-	 * @param string $timezone
 	 *
 	 * @return int
 	 */
-	protected static function get_event_timestamp( $event_id, $type = 'Start', $timezone = null ) {
+	protected static function get_event_timestamp( $event_id, $type = 'Start' ) {
```

Comment:
You're eliminating the `$timezone` parameter - but you haven't updated this any place else. For example, `tribe_get_start_time()` also accepts a `$timezone` param and it would ultimately be passed along the chain to this one.

* If we're sure that param isn't needed (we need to think about 3rd party code, too) we should update all other related code accordingly
* We should consider if some sort of formal deprecation would make sense